.github/workflows/scorecards.yml

```diff
@@ -11,7 +11,7 @@ on:
     branches: ["main"]
 
 # Declare default permissions as read only.
-permissions: read-all
+permissions: {}
```

Comment:
This might cause scorecard to fail, then we should figure out what exactly is needed. See https://github.com/ossf/scorecard-action/issues/1461

Reply:
Hmmm, yeah.  I guess we can just try and see.  Thanks Matti.

Reply:
Seems to work without those permisssions https://github.com/numpy/numpy/actions/workflows/scorecards.yml. I am not sure where we see the results of the scan?

Reply:
there is a tag in the readme: https://securityscorecards.dev/viewer/?uri=github.com/numpy/numpy

Reply:
Cool. The info there indicates it ran on the commit from merging this PR, so I guess the permissions are OK.